integration-tests/integration/whereabouts/selectResidentialLocation.spec.js

```diff
@@ -1,16 +1,5 @@
 const moment = require('moment')
 
-// copy of { getCurrentPeriod } from backend/utils/.js
```

Comment:
This is duplicated in /selectActivityLocation.spec.js  too

Reply:
Argh - I did the hard work and left the unused function in.. I'll commit that now